tests/nested/core/coreconfig-services/task.yaml

```
@@ -0,0 +1,21 @@
+summary: Disable and enable back core services via snap set with reboot.
+
+systems: [ubuntu-18.04-64]
```

couldn't you just make this

```suggestion
systems: [ubuntu-18.04-64, ubuntu-20.04-64]
```

and then this test will run for uc20 and uc18? then we don't need to duplicate the same file. I guess I thought that tests/nested/core20 was for things that were uc20 specific, but if a test doesn't need to be uc20 specific then it can be run on uc20 here too (though certainly correct me if I'm wrong)

also why don't we run this with uc16 in the nested environment?

---

IIRC we do not use systemd-resolved in UC16, that was still resolveconf back in the day. But I agree about resuing the file if that is possible

---

For some reason in spread.yaml we have separate suites, and for tests/nested/core we have:
`systems: [ubuntu-16.04-64, ubuntu-18.04-64]` so this wouldn't work without changing spread.yaml.

From the quick scan the setup for tests/nested/core and tests/nested/core20 looks similar (if not identical) but maybe there are deeper reasons for the split; CC @sergiocazzolato

---

Thanks for looking into this! We can always do that in a followup, it's fine for now as long as we have it on the radar.

---

ah thanks, I forgot that systemd-resolved is not a thing in uc16... 
also regarding not duplicating these test files, I think that's fine to do in a followup if it requires changes to the spread.yaml, etc.

---

@stolowski @anonymouse64 
The main differences between core and core20 suites are:
 . default systems to run the suite
 . core20 uses NESTED_ENABLE_TPM=true and NESTED_ENABLE_SECURE_BOOT=true
So if you run a test for uc20 in core suite, it won't use neither secure boot nor tpm.